src/controllers/movie.js

```diff
@@ -0,0 +1,73 @@
+const { getMovieListDb, createMovieDb, getMovieByIdDb, updateMovieDb, getMovieListGtLtDb, getMovieListGtDb, getMovieListLtDb, createMovieAndScreeningDb, getMovieByTitleDb } = require('../domains/movie.js')
+
+// GET MOVIE LIST
+const getMovieList = async (req, res) => {
+    if (req.query) {
+        const runtimeLt = Number(req.query.runtimeLt)
+        const runtimeGt = Number(req.query.runtimeGt)
+        
+        if (runtimeLt && runtimeGt) {
+            const movieList = await getMovieListGtLtDb(runtimeLt, runtimeGt)
+            return res.status(200).json({ movies: movieList })
+        }
+        if (runtimeLt) {
+            const movieList = await getMovieListLtDb(runtimeLt)
+            return res.status(200).json({ movies: movieList })
+        }
+        if (runtimeGt) {
+            const movieList = await getMovieListGtDb(runtimeGt)
+            return res.status(200).json({ movies: movieList })
+        }
+    }
+
+    const movieList = await getMovieListDb()
+    return res.status(200).json({ movies: movieList })
+}
+
+// CREATE MOVIE
+const createMovie = async (req, res) => {
+    const { title, runtimeMins, screenings } = req.body
+
+    if (!title || !runtimeMins)
+    return res.status(400).json({ error: "Missing fields in the request body, please enter the film title and runtime in minutes."})
```

Comment:
I would try to make the error messages a little more generic so you can put them into a single place to avoid having strings everywhere